CORE_DEVELOPER.txt

```diff
@@ -0,0 +1,164 @@
+Core Contributor Guide
+======================
+
+Welcome, new core developer!  The core team appreciate the quality of
+your work, and enjoy working with you; we have therefore invited you
+to join us.  Thank you for your numerous contributions to the project
+so far.
+
+This document offers guidelines for your new role.  First and
+foremost, you should familiarize yourself with the project's
+`mission, vision, and values
+<https://github.com/scikit-image/scikit-image/pull/3585>`__.  When in
+doubt, always refer back here.
+
+As a core team member, you gain the responsibility of shepherding
+other contributors' through the review process; here are some
+guidelines.
+
+All Contributors Are Treated Similarly
```

Comment:
```suggestion
All Contributors Are Treated The Same
```

? "Similar" sounds too fuzzy to me. It's like, "similarly, but if you like Harry, you can give him special treatment".